middlewares/validators/user.js

```diff
@@ -52,9 +52,6 @@ const updateUser = async (req, res, next) => {
         .optional(),
       discordId: joi.string().optional(),
       roles: joi.object().keys({
-        archived: joi.boolean().required(),
-        in_discord: joi.boolean().required(),
-        developer: joi.boolean().optional(),
```

Comment:
this validator function is being used at multiple places. Are you sure removing here will not affect other places ?

Reply:
It is not being used in any user route apart from this